src/components/PrintModal/PrintModal.js

```diff
@@ -293,11 +305,24 @@ class PrintModal extends React.PureComponent {
                       name="annotations"
                       label={t('option.print.includeAnnotations')}
                       disabled={isPrinting}
-                      onChange = {() => this.setState(state => ({ includeAnnotations: !state.includeAnnotations }))}
-                      checked = {includeAnnotations}
+                      onChange={() =>
+                        this.setState(state => ({
+                          includeAnnotations: !state.includeAnnotations
+                        }))
+                      }
+                      checked={includeAnnotations}
                       center
                     />
                   </form>
+                  <div>
+                    <div className="col">
+                      <div>{`${t('option.print.pageQuality')}:`}</div>
+                      <select className="printQualitySelect" onChange={e => this.props.setPrintQuality(Number(e.target.value))}>
```

Comment:
Sorry one more thing that I noticed is that we probably want to set the `value` to be `this.props.printQuality`.

The reason is that clients are able to call `instance.setPrintQuality(x)` to set the print quality programmatically. In this case when they pass `2`, we want the modal to display `High`, not `Normal`.

If we set the `value` then we need to handle the case where clients pass in a number that's bigger than `2` to `instance.setPrintQuality`. In this case we want to display `High`, not `Normal`. I found one solution that works in Chrome is to move `<option value="2"` above `<option value="1"`... Not sure if it's a general solution.

Reply:
I see your concern.
we only want to support printQuality between 1~2 from the UI, so if user set any value programmatically larger than 2, then we should always display "high'.
Move the `<option value="2"` will still display in this case. (Tested in Chrome, FF, Edge on Windows)